labonneboite/importer/jobs/extract_etablissements.py

```diff
@@ -16,6 +16,16 @@
 from labonneboite.importer.jobs.base import Job
 from labonneboite.importer.jobs.common import logger
 
+# This list contains siret that must not be found in datas,
```

Comment:
data

Reply:
https://forum.wordreference.com/threads/data-or-datas.11171/

Reply:
Data is the plural form of datum. However, although controversial, data is becoming acceptable as both singular and plural.

Reply:
Done